packages/lib-utils/src/k8s/hooks/k8s-watcher.ts

```diff
@@ -77,7 +77,7 @@ export const getReduxData = (immutableData, resource: WatchK8sResource) => {
 };
 
 export const getWatchData: GetWatchData = (resource, k8sModel, cluster) => {
-  if (!k8sModel || !resource || !resource.name || !resource.namespace) {
+  if (!k8sModel || !resource?.namespace) {
```

Comment:
This would prevent us from watching resources that are not namespaced, correct? Is that expected? I know this is also currently the case so it's not a comment on this PR but a general question.